app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -116,6 +150,38 @@ class EditProfileFragment: DialogFragment() {
         profileViewModel.successfulUpdate.value = null
     }
 
+    private fun validateDate(dateOfBirth: String): Boolean {
+        var isValid: Boolean
+        if (dateOfBirth == "null") {
```

Comment:
`"null"` or `null`? @m-murad @annabauza

Reply:
`null`

Reply:
ahh I understand why is that, sometimes when you map null to String it comes as "null". I think we should try to parse to `toBooleanOrNull` I think I saw this method in Kotlin.